docs/source/index.md

```diff
@@ -1,6 +1,6 @@
 # MarketPlace Docs
 
-Welcome to the [MarketPlace](https://www.materials-marketplace.eu/) documentation!
+Welcome to the [MarketPlace](https://www.materials-marketplace.eu/) documentation (version {platform_version})!
```

Comment:
I think it doesn't look so good. Maybe we mention the version in  a separate line or even in the footer?

Reply:
Do you think an admonition at the top (like info or so) would be better?
It seems there might be [some issues](https://github.com/executablebooks/jupyter-book/issues/1660) with templates, and we cannot customise the footer.

Reply:
Yeah, maybe that could work better.

Reply:
Ok, I applied the changes. Feel free to re-review.